setup.cfg

```diff
@@ -0,0 +1,17 @@
+[options.entry_points]
+console_scripts =
+    fio = fiona.fio.main:main_group
+fiona.fio_commands =
+    bounds = fiona.fio.bounds:bounds
+    calc = fiona.fio.calc:calc
+    cat = fiona.fio.cat:cat
+    collect = fiona.fio.collect:collect
+    distrib = fiona.fio.distrib:distrib
+    dump = fiona.fio.dump:dump
+    env = fiona.fio.env:env
+    filter = fiona.fio.filter:filter
+    info = fiona.fio.info:info
+    insp = fiona.fio.insp:insp
+    load = fiona.fio.load:load
+    ls = fiona.fio.ls:ls
+    rm = fiona.fio.rm:rm
```

Comment:
This duplicates config in pyproject.toml but is necessary. Without a setup.cfg we can't do `python setup.py develop` (via pip). And that's the only way we can have fast local tests with `make dockertest`.